.travis.yml

```diff
@@ -132,6 +132,11 @@ jobs:
     - cd $TRAVIS_BUILD_DIR/examples/cryptocurrency/examples && ./test.sh
     - cd $TRAVIS_BUILD_DIR/examples/cryptocurrency/examples && ./test.ps1
 
+  # Additional tests
+  - env: FEATURE=test-aux
+    script:
+    - cd $TRAVIS_BUILD_DIR/exonum && cargo test --features doctests --doc explorer
```

Comment:
This is needed for heavyweight blockchain initialization, which is shared among all explorer doctests. I would love to get rid of it, but don't know how.